### PR TITLE
replace: jira.appcelerator.org -> jira-archive.titaniumsdk.com

### PR DESCRIPTION
**Pull request by caspahouzer**

### CODE REVIEW

File: docs/guide/Titanium_SDK/Titanium_SDK_Getting_Started/Installation_and_Configuration/Installing_Platform_SDKs/Installing_the_Android_SDK.md [hunk 1]
```diff
@@ -195,7 +195,7 @@ To install Android SDK on Windows:
 
 6. Once the requested components have been installed, click **Finish** button.
 
-The following steps are the normal process to install an Android SDK with Axway Appcelerator Studio (Appc Studio) but due to the bug noted in [TISTUD-9035](https://jira.appcelerator.org/browse/TISTUD-9035), Studio cannot install Android SDKs at this time.
+The following steps are the normal process to install an Android SDK with Axway Appcelerator Studio (Appc Studio) but due to the bug noted in [TISTUD-9035](https://jira-archive.titaniumsdk.com/TISTUD-9035), Studio cannot install Android SDKs at this time.
```
Comment (cb1kenobi) [inline docs/guide/Titanium_SDK/Titanium_SDK_Getting_Started/Installation_and_Configuration/Installing_Platform_SDKs/Installing_the_Android_SDK.md]: We never grabbed TISTUD tickets. Besides, this sentence is no longer relevant.

File: docs/guide/Titanium_SDK/Titanium_SDK_Guide/Appendices/tiapp.xml_and_timodule.xml_Reference.md [hunk 2]
```diff
@@ -941,7 +941,7 @@ Because images are store in the Asset Catalog, reading image files manually from
 ::: danger ❗️ Experimental Feature
 This is an experimental feature with known issues:
 
-* [TIMOB-19806](https://jira.appcelerator.org/browse/TIMOB-19806)
+* [TIMOB-19806](https://jira-archive.titaniumsdk.com/TIMOB-19806)
```
Comment (cb1kenobi) [inline docs/guide/Titanium_SDK/Titanium_SDK_Guide/Appendices/tiapp.xml_and_timodule.xml_Reference.md]: This ticket does not exist. I have no idea if it was private or deleted or what, but we don't have it.

So, I say remove the entire `:::` block.

File: docs/guide/Titanium_SDK/Titanium_SDK_Guide/Atom_Package/Atom_Package_Release_Notes/Atom_Package_1.1.0_Release_Note.md [hunk 3]
```diff
@@ -9,7 +9,7 @@ Atom Package 1.1.0 is a minor release that includes new features, improvements,
 
 ## New features
 
-* [ATOM-11](https://jira.appcelerator.org/browse/ATOM-11) - Snippets
+* [ATOM-11](https://jira-archive.titaniumsdk.com/ATOM-11) - Snippets
```
Comment (cb1kenobi) [inline docs/guide/Titanium_SDK/Titanium_SDK_Guide/Atom_Package/Atom_Package_Release_Notes/Atom_Package_1.1.0_Release_Note.md]: We don't have ATOM tickets. Didn't even know there was an ATOM project in JIRA. Oops. Just leave the `ATOM-11` text and remove the link. Repeat for all ATOM-* links.

File: docs/guide/Titanium_SDK/Titanium_SDK_Guide/Atom_Package/Atom_Package_Release_Notes/README.md [hunk 4]
```diff
@@ -11,7 +11,7 @@ Atom Package 1.1.0 is a minor release that includes new features, improvements,
 
 ### New features
 
-* [ATOM-11](https://jira.appcelerator.org/browse/ATOM-11) - Snippets
+* [ATOM-11](https://jira-archive.titaniumsdk.com/ATOM-11) - Snippets
```
Comment (cb1kenobi) [inline docs/guide/Titanium_SDK/Titanium_SDK_Guide/Atom_Package/Atom_Package_Release_Notes/README.md]: Same in this file... ATOM does not exist, remove link

**Comment by m1ga** [comment]
minor issue:
we don't have the ATOM issues like: https://jira-archive.titaniumsdk.com/ATOM-11 (16 links only) so perhaps remove the link so we don't have 404 right away

**Comment by caspahouzer** [comment]
Will they never appear? Then they can be removed

File: docs/guide/Titanium_SDK/Titanium_SDK_Guide/Contributing_to_Titanium/Contributing_to_Open_Source_Titanium_Modules.md [hunk 5]
```diff
@@ -19,7 +19,7 @@ Thus, you must first electronically sign the [CLA](http://developer.appcelerator
 
 ## Create a Ticket
 
-If you are having an issue with an open source module, please create a ticket for it under the [TIMODOPEN](https://jira.appcelerator.org/browse/TIMODOPEN) project. Even if you are going to fix the issue yourself and submit a pull request, it is important to have a ticket so we can track the issue. When creating a ticket, be sure to add sample code and instructions that can be used to reproduce the issue.
+If you are having an issue with an open source module, please create a ticket for it under the [TIMODOPEN](https://jira-archive.titaniumsdk.com/TIMODOPEN) project. Even if you are going to fix the issue yourself and submit a pull request, it is important to have a ticket so we can track the issue. When creating a ticket, be sure to add sample code and instructions that can be used to reproduce the issue.
```
Comment (cb1kenobi) [inline docs/guide/Titanium_SDK/Titanium_SDK_Guide/Contributing_to_Titanium/Contributing_to_Open_Source_Titanium_Modules.md]: Well, no more TIMODOPEN JIRA project, so replace the link with:

```
... please create a ticket in that module's repository issue tracker.
```

**Comment by caspahouzer** [comment]
But then I also have to remove the complete Atom release notes?

**Review by cb1kenobi — CHANGES_REQUESTED** [review]

File: docs/guide/Titanium_SDK/Titanium_SDK_Guide/Atom_Package/Atom_Package_Release_Notes/README.md (hunk 4 above)
Comment (caspahouzer) [reply docs/guide/Titanium_SDK/Titanium_SDK_Guide/Atom_Package/Atom_Package_Release_Notes/README.md]: links removed

File: docs/guide/Titanium_SDK/Titanium_SDK_Guide/Appendices/tiapp.xml_and_timodule.xml_Reference.md [hunk 6]
```diff
@@ -938,12 +938,7 @@ Because images are store in the Asset Catalog, reading image files manually from
...
 **Since Release 5.1.0**. Enables the iOS [auto-layout engine](https://developer.apple.com/library/ios/documentation/UserExperience/Conceptual/AutolayoutPG/index.html#//apple_ref/doc/uid/TP40010853) to layout the children components of a view. Disabled by default.
 
-::: danger ❗️ Experimental Feature
-This is an experimental feature with known issues:
-
-* [TIMOB-19806](https://jira.appcelerator.org/browse/TIMOB-19806)
-
-If you find any issue using this feature, file a bug in [JIRA](https://jira.appcelerator.org/).
+If you find any issue using this feature, file a bug in [GitHub Issues](https://github.com/tidev/titanium_mobile/issues).
 :::
```
Comment (cb1kenobi) [inline docs/guide/Titanium_SDK/Titanium_SDK_Guide/Appendices/tiapp.xml_and_timodule.xml_Reference.md]: Eh, just remove this sentence as it relates to it being experimental. Don't forget the dangling `:::`.

**Review by cb1kenobi — CHANGES_REQUESTED** [review]

**Comment by caspahouzer** [comment]
All done

File: docs/guide/Titanium_SDK/Titanium_SDK_Guide/Appendices/tiapp.xml_and_timodule.xml_Reference.md [hunk 7]
```diff
@@ -938,13 +938,7 @@ Because images are store in the Asset Catalog, reading image files manually from
...
 **Since Release 5.1.0**. Enables the iOS [auto-layout engine](https://developer.apple.com/library/ios/documentation/UserExperience/Conceptual/AutolayoutPG/index.html#//apple_ref/doc/uid/TP40010853) to layout the children components of a view. Disabled by default.
 
-::: danger ❗️ Experimental Feature
-This is an experimental feature with known issues:
-
-* [TIMOB-19806](https://jira.appcelerator.org/browse/TIMOB-19806)
-
-If you find any issue using this feature, file a bug in [JIRA](https://jira.appcelerator.org/).
-:::
+If you find any issue using this feature, file a bug in [GitHub Issues](https://github.com/tidev/titanium_mobile/issues).
```
Comment (cb1kenobi) [inline docs/guide/Titanium_SDK/Titanium_SDK_Guide/Appendices/tiapp.xml_and_timodule.xml_Reference.md]: Sorry, what I'm trying to say is we don't need this sentence. It's related to an experimental feature that has been apart of the SDK for years, so I doubt it's "experimental" these days. Saying if you find any issues, file a bug is a given. We don't have to restate the obvious.

**Review by cb1kenobi — CHANGES_REQUESTED** [review]
Getting close! Just need to fix the merge conflicts and we're good.

**Comment by caspahouzer** [comment]
I don't know how to solve this merge conflicts. Locally, I have no conflicts. I've merged master into this PR and pushed and it seems all good

**Comment by m1ga** [comment]
Those are the files you've change in: https://github.com/tidev/titanium-docs/pull/99
since it is the same line (you've change the github url in the other one, the jira ticket in this one) it doesn't know which one it should use.
You have to update your master branch locally with the github master branch or merge your pr_replace_appc_git_url into your local master. After that try to merge master into pr_replace_jira and it should show the same conflicts.

**Comment by caspahouzer** [comment]
@cb1kenobi all fine now. can you please review your requests?

File: docs/guide/Titanium_SDK/Titanium_SDK_Guide/Contributing_to_Titanium/How_to_Report_a_Bug_or_Make_a_Feature_Request/README.md [hunk 8]
```diff
@@ -31,12 +31,12 @@ To help organize the information, every JIRA ticket is assigned to one of a smal
...
-| [Appcelerator Studio](http://jira.appcelerator.org/browse/TISTUD) | TISTUD | ![check](/images/guide/s/en_GB/5637/e1ef10868e8fe2f234a1a0b171b01cde1d9717c4.31/_/images/icons/emoticons/check.png) | ![error](/images/guide/s/en_GB/5637/e1ef10868e8fe2f234a1a0b171b01cde1d9717c4.31/_/images/icons/emoticons/error.png) | ![check](/images/guide/s/en_GB/5637/e1ef10868e8fe2f234a1a0b171b01cde1d9717c4.31/_/images/icons/emoticons/check.png) |
+| [Aptana Studio](https://jira-archive.titaniumsdk.com/APSTUD) | APSTUD | ![check](/images/guide/s/en_GB/5637/e1ef10868e8fe2f234a1a0b171b01cde1d9717c4.31/_/images/icons/emoticons/check.png) | ![error](/images/guide/s/en_GB/5637/e1ef10868e8fe2f234a1a0b171b01cde1d9717c4.31/_/images/icons/emoticons/error.png) | ![check](/images/guide/s/en_GB/5637/e1ef10868e8fe2f234a1a0b171b01cde1d9717c4.31/_/images/icons/emoticons/check.png) |
```
Comment (cb1kenobi) [inline docs/guide/Titanium_SDK/Titanium_SDK_Guide/Contributing_to_Titanium/How_to_Report_a_Bug_or_Make_a_Feature_Request/README.md]: No APSTUD project, so just remove the link.

File: docs/guide/Titanium_SDK/Titanium_SDK_Guide/Contributing_to_Titanium/How_to_Report_a_Bug_or_Make_a_Feature_Request/README.md [hunk 9]
```diff
@@ -31,12 +31,12 @@ To help organize the information, every JIRA ticket is assigned to one of a smal
...
+| [Alloy](https://jira-archive.titaniumsdk.com/ALOY) | ALOY | ![check](/images/guide/s/en_GB/5637/e1ef10868e8fe2f234a1a0b171b01cde1d9717c4.31/_/images/icons/emoticons/check.png) | ![error](/images/guide/s/en_GB/5637/e1ef10868e8fe2f234a1a0b171b01cde1d9717c4.31/_/images/icons/emoticons/error.png) | ![check](/images/guide/s/en_GB/5637/e1ef10868e8fe2f234a1a0b171b01cde1d9717c4.31/_/images/icons/emoticons/check.png) |
+| [Appcelerator Studio](https://jira-archive.titaniumsdk.com/TISTUD) | TISTUD | ![check](/images/guide/s/en_GB/5637/e1ef10868e8fe2f234a1a0b171b01cde1d9717c4.31/_/images/icons/emoticons/check.png) | ![error](/images/guide/s/en_GB/5637/e1ef10868e8fe2f234a1a0b171b01cde1d9717c4.31/_/images/icons/emoticons/error.png) | ![check](/images/guide/s/en_GB/5637/e1ef10868e8fe2f234a1a0b171b01cde1d9717c4.31/_/images/icons/emoticons/check.png) |
```
Comment (cb1kenobi) [inline docs/guide/Titanium_SDK/Titanium_SDK_Guide/Contributing_to_Titanium/How_to_Report_a_Bug_or_Make_a_Feature_Request/README.md]: No TISTUD project, so just remove the link.

File: docs/guide/Titanium_SDK/Titanium_SDK_Guide/Contributing_to_Titanium/How_to_Report_a_Bug_or_Make_a_Feature_Request/README.md [hunk 10]
```diff
@@ -111,7 +111,7 @@ A good read on this is Eric Lippert's [How to debug small programs](http://ericl
 
 Assuming you have consulted the community via the Q&A, and you have your [test case](#CreatingaTestCase) to hand, you are ready to create a ticket:
 
-* open the JIRA [Titanium Community](http://jira.appcelerator.org/browse/TC) project page. See the [#Projects Overview](#ProjectsOverview) section for more information about the projects.
+* open the JIRA [Titanium Community](https://jira-archive.titaniumsdk.com/TC) project page. See the [#Projects Overview](#ProjectsOverview) section for more information about the projects.
```
Comment (cb1kenobi) [inline docs/guide/Titanium_SDK/Titanium_SDK_Guide/Contributing_to_Titanium/How_to_Report_a_Bug_or_Make_a_Feature_Request/README.md]: I don't think there's ever been a TC project. I've always known it to be AC.

Anyways, this link needs to be https://github.com/tidev/titanium_mobile/issues and update the wording from JIRA to GitHub issue.

File: docs/guide/Titanium_SDK/Titanium_SDK_Guide/Contributing_to_Titanium/Platform_Development/Pull_Request_Guide.md [hunk 11]
```diff
@@ -55,7 +55,7 @@ All pull requests that get integrated into the Titanium SDK have an associated J
 
 1. Find an existing Titanium Mobile (TIMOB) ticket with which to associate your ticket. You can search the TIMOB project for an existing feature request or bug with the following link: [Titanium Mobile (TIMOB) Jira project](http://jira.appcelerator.org/secure/IssueNavigator.jspa)
 
-2. Find or create a new ticket in the Titanium Community (TC) Jira project. Community members have the ability to create new tickets here, unlike the TIMOB project. If you want to add a feature or fix a bug that has not yet been logged in either Jira project, you can do so at the following link: [Titanium Community (TC) Jira project](http://jira.appcelerator.org/browse/TC)
+2. Find or create a new ticket in the Titanium Community (TC) Jira project. Community members have the ability to create new tickets here, unlike the TIMOB project. If you want to add a feature or fix a bug that has not yet been logged in either Jira project, you can do so at the following link: [Titanium Community (TC) Jira project](https://jira-archive.titaniumsdk.com/TC)
```
Comment (cb1kenobi) [inline docs/guide/Titanium_SDK/Titanium_SDK_Guide/Contributing_to_Titanium/Platform_Development/Pull_Request_Guide.md]: Same here, use https://github.com/tidev/titanium_mobile/issues.

**Comment by cb1kenobi** [comment]
@caspahouzer There's a couple of places it says to create a ticket in JIRA when it should be GitHub Issues: https://github.com/tidev/titanium_mobile/issues. I highlighted a few, but maybe there are others.

**Review by cb1kenobi — CHANGES_REQUESTED** [review]

File: docs/guide/Titanium_SDK/Titanium_SDK_Guide/Contributing_to_Titanium/How_to_Report_a_Bug_or_Make_a_Feature_Request/README.md (hunk 10 above)
Comment (caspahouzer) [reply docs/guide/Titanium_SDK/Titanium_SDK_Guide/Contributing_to_Titanium/How_to_Report_a_Bug_or_Make_a_Feature_Request/README.md]: There are tons of JIRA mentions. That should be handled in another PR

Comment (cb1kenobi) [reply docs/guide/Titanium_SDK/Titanium_SDK_Guide/Contributing_to_Titanium/How_to_Report_a_Bug_or_Make_a_Feature_Request/README.md]: Sorry, yes there are tons of JIRA mentions, but in this case, I'm specifically referring to this link. We don't want to say create a JIRA ticket by going to GitHub issues.

Comment (caspahouzer) [reply docs/guide/Titanium_SDK/Titanium_SDK_Guide/Contributing_to_Titanium/How_to_Report_a_Bug_or_Make_a_Feature_Request/README.md]: Fixed that

**Review by cb1kenobi — APPROVED** [review]
LGTM! Thanks @caspahouzer!

**Pull request merged**